packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -1444,8 +1444,9 @@ export function createProjectionNode<I>({
             this.pendingAnimation = sync.update(() => {
                 globalProjectionState.hasAnimatedSinceResize = true
 
-                this.currentAnimation = animate(0, animationTarget, {
+                this.currentAnimation = animateValue({
```

Comment:
Switching from the full `animate()` function to the specific animator we want (main thead value animator) to avoid bundling in all the new selector weight